config/initializers/flipper.rb

```diff
@@ -15,7 +24,19 @@
   end
 end
 
-Flipper.enable(:project_comments)
+# set default flipper configuration
+if !Rails.env.test? then
+  client = Redis.new
+  enabled_features = client.smembers("flipper_features")
```

Comment:
`Flipper.features`

Reply:
Ah I was looking for `Flipper.features`, for some reason I couldn't find it in the documentation easily.